src/services/workflow.js

```diff
@@ -94,11 +94,27 @@ function start ({ communication, serverPort }) {
     if (populateLogs) {
       const queryBuilder = WorkflowLog.query().where({ workflowId: workflow.id })
       const actionCountQueryBuilder = queryBuilder.clone()
-        .whereIn('type', ['action'])
-        .whereJsonNotSupersetOf('step', { stopped: true })
-        .whereJsonNotSupersetOf('step', { skipped: true })
-        .whereJsonNotSupersetOf('step', { error: true, handleErrors: false })
-      const startedACountQueryBuilder = queryBuilder.clone().whereIn('type', ['action'])
+        .where(qb => {
+          return qb
+            .where(qb2 => {
+              return qb2
+                .whereIn('type', ['action'])
+
+                // DEPRECATED: old step object properties
+                .whereJsonNotSupersetOf('step', { stopped: true })
+                .whereJsonNotSupersetOf('step', { skipped: true })
+                .whereJsonNotSupersetOf('step', { error: true, handleErrors: false })
+                // DEPRECATED:END
+            })
+            .orWhere(qb2 => {
+              return qb2
+                .where('type', 'runError')
+                .whereJsonSupersetOf('step', { handleErrors: true })
+            })
+        })
+      const startedACountQueryBuilder = queryBuilder.clone().whereIn('type',
+        ['action', 'skipped', 'stopped', 'runError']
```

Comment:
Why not everything _but_ `notification` type to ensure we don’t forget anything?

Reply:
Indeed there are many types. It is everything but `notification` and `preRunError`. I'll update with a `whereNotIn` with those two types, thanks.